doc/config.md

```diff
@@ -28,7 +28,7 @@ Example:
 
 ## Environment Variables
 
-The application will look for environment variables that are able to override the configuration defined in the `conf.toml` file. These environment variables are prefixed with `VTT` and follow this pattern `VTT_<conf value>`. e.g. `VTT_LISTENPORT`.
+The application will look for environment variables that are able to override the configuration defined in the `conf.toml` file. These environment variables are prefixed with `VTT` and follow this pattern `VTT_<conf value>`. e.g. `VTT_LISTENPORT`. As per the vyper documentation the environment variables will be lookup as uppercase letters. e.g DbHost will be lookup as DBHOST, so the environment variable should be set as VTT_DBHOST.
```

Comment:
It is a valid point about the uppercase letters being required for the environment variables, Maybe you should expand on the `VTT_LISTENPORT` example rather than adding a second one, and link to the vyper documentation.